migrations/sql/V2019.04.08.09.32__create_incident_reports.sql

```diff
@@ -0,0 +1,48 @@
+CREATE TABLE IF NOT EXISTS mine_incident_followup_type
+(
+    mine_incident_followup_type_code                character varying(3) PRIMARY KEY,
+    description                                     character varying(100),
+    display_order                                   integer NOT NULL,
+    active_ind                                      boolean NOT NULL DEFAULT 'true',
+
+    create_user                                     character varying(60)                    NOT NULL,
+    create_timestamp                                timestamp with time zone DEFAULT now()   NOT NULL,
+    update_user                                     character varying(60)                    NOT NULL,
+    update_timestamp                                timestamp with time zone DEFAULT now()   NOT NULL
+);
+ALTER TABLE mine_incident_followup_type OWNER TO mds;
+COMMENT ON TABLE mine_incident_followup_type IS 'lookup table for the types of EMPR reactions to a given mine_incident.';
+
+CREATE TABLE IF NOT EXISTS mine_incident
+(
+    mine_incident_id                serial                      PRIMARY KEY,
+    mine_incident_id_year           integer                     NOT NULL,
+    mine_incident_guid              uuid DEFAULT gen_random_uuid()   NOT NULL,
```

Comment:
What's the reason for a guid and an id?

Reply:
We never want to expose the serial id, used for FK's, anything that needs to be uniquely identified from the outside needs a guid.

Reply:
Makes sense 👍